README.md

```diff
@@ -63,8 +63,90 @@ Check the output
 
 * Locate the output on disk at `~/aircan-example-1.json`
 
+## Using Aircan DAGs
 
-### Example 2: Local file to CKAN DataStore
+
+### Example 2: Local file to CKAN DataStore using the Datastore API
+
+#### Preliminaries: Setup your CKAN instance
+
+We'll assume you have:
+
+* a local CKAN setup and running at https://localhost:5000
+* a dataset (for example, `my-dataset`) with a resource (for example, `my-resource` with the `my-res-id-123` as its `resource_id`).
```

Comment:
For really nice quickstart we would provide them with a script to do that - i think this was in original README. But it is kind of a bonus.